### PR TITLE
Tw 780 savings page kord fi

### PR DESCRIPTION
**Pull request by herkoss**

### CODE REVIEW

File: src/apis/kord-fi/index.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+import { fallbackTezosToolkit } from '../youves/utils';
+import { KordFiLendStats, KordFiStatsResponse, KordFiUserDepositsResponse } from './types';
+
+const kordFiApi = axios.create({
+  baseURL: 'https://back-llb-beta.kord.fi'
+});
+
+const TEZOS_CONTRACT_ADDRESS = 'KT19qWdPBRtkWrsQnDvVfsqJgJB19keBhhMX';
+const TZBTC_CONTRACT_ADDRESS = 'KT1WL6sHt8syFT2ts7NCmb5gPcS2tyfRxSyi';
+const TEZOS_TOKEN: EarnOpportunityToken = {
```
Comment (keshan3262) [inline src/apis/kord-fi/index.ts:25]: Sorry, could you create a function like `toEarnOpportunityToken` in `src/utils/earn.utils.ts` and use it for both tokens?

File: src/modals/manage-earn-opportunity-modal/stake-form/constants.ts [hunk 2]
```diff
@@ -28,5 +28,6 @@ export const earnOpportunitiesRisksPoints: Record<EarnOpportunityTypeEnum, strin
   [EarnOpportunityTypeEnum.YOUVES_STAKING]: youvesSavingsRisksPoints,
   [EarnOpportunityTypeEnum.DEX_TWO]: quipuswapFarmsRisksPoints,
   [EarnOpportunityTypeEnum.STABLESWAP]: quipuswapFarmsRisksPoints,
-  [EarnOpportunityTypeEnum.LIQUIDITY_BAKING]: liquidityBakingRisksPoints
+  [EarnOpportunityTypeEnum.LIQUIDITY_BAKING]: liquidityBakingRisksPoints,
+  [EarnOpportunityTypeEnum.KORD_FI_SAVING]: [''] // Add warning text
```
Comment (keshan3262) [inline src/modals/manage-earn-opportunity-modal/stake-form/constants.ts:32]: I hope @soulofthenation will provide it for us

File: src/store/savings/epics.ts [hunk 3]
```diff
@@ -58,31 +59,42 @@ const loadAllSavingsItemsAndStakes: Epic = (action$: Observable<Action>, state$:
...
+        getKordFiUserDeposits$(selectedAccount.publicKeyHash)
+      ]).pipe(
+        map(([youvesStakesEntries, kordFiStakesEntries]) => {
+          return {
+            ...Object.fromEntries(
+              youvesStakesEntries.filter((entry): entry is [string, UserStakeValueInterface] => isDefined(entry[1]))
+            ),
+            ...kordFiStakesEntries
+          };
+        }),
```
Comment (keshan3262) [inline src/store/savings/epics.ts:91]:
```suggestion
        map(([youvesStakesEntries, kordFiStakesEntries]) => ({
          ...Object.fromEntries(
            youvesStakesEntries.filter((entry): entry is [string, UserStakeValueInterface] => isDefined(entry[1]))
          ),
          ...kordFiStakesEntries
        })),
```
If you think that it's less convenient, you may resolve this discussion silently

**Review by keshan3262 — COMMENTED** [review]
Please fix ts errors

**Review by lendihop — APPROVED** [review]

**Review by keshan3262 — APPROVED** [review]

**Review by NovakVlad2424 — APPROVED** [review]
ios done: 1.15_923

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_923

**Review by lendihop — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.16_998

**Review by NovakVlad2424 — APPROVED** [review]
ios done: 1.16_998

**Pull request merged**